packages/sysutils/ethmactool/scripts/ethmactool-config

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+COMPATIBLE=$(/usr/bin/dtsoc)
+MAC_STEP=""
+
+validate_mac() {
+  [ ${#MAC} -eq 12 -a "${MAC}" != "000000000000" ]
+}
+
+fixup_self_mac() {
+  # clear multicast bit and set local assignment bit (IEEE802)
+  MAC=$(printf '%X\n' "$(( (0x$MAC & 0xFEFFFFFFFFFF) | 0x020000000000 ))")
```

Comment:
`$(( (0x$MAC & 0xFEFFFFFFFFFF) | 0x020000000000 ))"`
is that `"$(( ... ))"`needed ?

Reply:
Yes, required for the calculation.